### PR TITLE
adding quick replies

### PR DESCRIPTION
**Pull request by MoisesFlores22**
What this PR does?
Adds the quick replies buttons

Any background context you want to provide?

🚧
Some buttons were displaying as cards without the  correct properties

GIF of the Work: 

apr-21-2017 15-13-59

 

- [x] @AlexisGlez 

- [x] @JoseILopez 

- [ ]  @rguinea 

- [ ] @ricardogrw 

- [ ] @castrosalazr

### CODE REVIEW

File: src/Attachment.tsx [hunk 1]
```diff
@@ -153,10 +153,19 @@ export const AttachmentView = (props: {
             e.stopPropagation();
         });
 
-    const buttons = (buttons: CardAction[]) => buttons &&
+    const buttons = (buttons: CardAction[]) => ( buttons &&
```
Comment (JoseILopez) [inline src/Attachment.tsx]: I am not sure about having a space after the opening parenthesis. It is a non-blocker though.

File: src/Attachment.tsx [hunk 2]
```diff
@@ -17,6 +17,21 @@ export interface QueryParams {
...
+        text?: string;
+        images?: {
+            url: string;
+        }[];
+        payload?:string;
+        buttons?: CardAction[];
+        tap?: CardAction;
+    };
+}
+
```
Comment (JoseILopez) [inline src/Attachment.tsx]: Double check this interface, as it is crashing on build.

Comment (MoisesFlores22) [inline src/Attachment.tsx]: @JoseILopez i think I fixed that :)

**Comment by JoseILopez** [comment]
Tested locally. It works now! 💯 🚀

File: wz-botchat.css [hunk 3]
```diff
@@ -317,6 +315,56 @@ ul.wc-card-buttons li button:last-child:active {
     color: white;
     border-radius: 0 0 8px 8px;
 }
+.wc-card.hero {
+    background-color: #ffffff; 
+    border: 1px solid #d2dde5;
+}
```
Comment (AlexisGlez) [inline wz-botchat.css]: These two properties are not aligned with the others.

File: wz-botchat.css [hunk 4]
```diff
@@ -397,6 +445,11 @@ ul.wc-card-buttons li button:last-child:active {
     width: 100px;
 }
 
+.wc-card.thumbnail {
+    background-color: #ffffff;
+    border: 1px solid #d2dde5;
```
Comment (AlexisGlez) [inline wz-botchat.css]: These two properties are not aligned with the others.

File: src/Attachment.tsx [hunk 5]
```diff
@@ -171,6 +180,7 @@ export const AttachmentView = (props: {
             if (!attachment.content)
                 return null;
             return (
+                attachment.content.payload ? quickReplies(attachment.content.buttons) :
```
Comment (AlexisGlez) [inline src/Attachment.tsx]: I think It'd be better if you split the ternary into more lines. For example:
```
attachment.content.payload
? quickReplies(attachment.content.buttons)
: <div>....
```
or even better, you could have a stateless component with that div and have it in a single line:
```
attachment.content.payload ? quickReplies(attachment.content.buttons) : <NewComponent />
```
what do you think?

**Comment by AlexisGlez** [comment]
LGTM 💯

**Pull request merged**